### ISSUE TITLE
Not compatible with IntelliJ IDEA 2024.1

### ISSUE THREAD

**Issue by pawelsa** [localizely/flutter-intl-intellij#117]
Current version does not support IntelliJ IDEA 2024.1

**Comment by lzoran** [comment]
Thank you for bringing this to our attention!

I've conducted a quick check and it seems that IntelliJ IDEA 2024.1 is reporting errors for deprecated code. Although the Flutter Intl plugin actions complete successfully, the IDE still catches exceptions.

Until we release a new version of the Flutter Intl plugin that fixes this issue, you might consider using a previous version of IntelliJ IDEA that is compatible with the Flutter Intl plugin. Alternatively, if these IDE errors do not affect your workflow, you may choose to ignore them.

```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.localizely.flutter.intl.ui.ExtractStringKeyDialog$1' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: com.localizely.flutter-intl]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:199)
```

**Comment by goranluledzija** [comment]
This is fixed in the latest plugin version 1.18.5-2023.2
Closing this issue now, but if @pawelsa still notices the issues feel free to open this issue again
